components/Banner.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <div class="banner">
+    <transition name="slide">
+      <div class="banner-container" v-if="open">
+        <p>{{ message }}</p>
+        <nacelle-button
+          class="nacelle-banner-button"
+          @click="goTo(buttonPath)"
+          color="secondary"
+          >{{ buttonText }}</nacelle-button
+        >
+        <img
+          @click="closeBanner"
+          class="close-banner"
+          src="/alertbar-x-icon.svg"
+        />
+      </div>
+    </transition>
+  </div>
+</template>
+
+<script>
+import NacelleButton from './NacelleButton.vue'
+export default {
+  components: {
+    NacelleButton
+  },
+  data() {
+    return {
+      open: false
+    }
+  },
+  props: {
+    title: {
+      type: String,
+      default: ''
+    },
+    buttonText: {
+      type: String,
+      default: ''
+    },
+    buttonPath: {
+      type: String,
+      default: ''
+    },
+    message: {
+      type: String,
+      default: ''
+    }
+  },
+  mounted() {
+    setTimeout(() => {
+      this.open = true
+    }, 1000)
+  },
```

Comment:
What's the reason behind the setTimeout?

Reply:
The timeout is set on the Nacelle website banner as well, I assume it's mostly just to get the little slide up animation on load?

Reply:
Makes sense to me!